TelCo.ColorCoder/ColorMapUtil.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    class ColorMapUtil

```

Comment:
Avoid names like `...Util`. It doesn't convey what a consumer of the class can expect in here, so maintainers will tend to use this to throw in all miscellaneous stuff. This class finally would become a 'big ball of mud'.
Name it as something the consumers would expect to do with it.